src/engine/matchers/string.js

```diff
@@ -0,0 +1,40 @@
+/**
+Copyright 2016 Split Software
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+**/
+'use strict';
+
+const log = require('../../utils/logger')('splitio-engine:matcher');
+
+function stringMatcherContext(ruleAttr /*: regex */) /*: Function */ {
+  return function stringMatcher(runtimeAttr /*: string */) /*: bool */ {
```

Comment:
shouldn't this be matchesStringMatcher ? Naming it only string seems like something that may be confusing down the line. And also it's not respecting the SDK Spec name. 

If we could update the name (with the necessary updates on the file name, require statements, etc) I think it would be easier to read down the line.